pages/index.tsx

```diff
@@ -1,16 +1,16 @@
+import type { NextPage } from 'next';
 import IntroPage from '../components/intropage/intropage';
 import Overview from '../components/Overview/overview';
 import Footer from '../components/Footer/footer';
 import Main from '../layout/Main';
 
-function Home() {
+const Home: NextPage = () => {
```

Comment:
I would be happy if you'll be sticking with the default code style. I'll update the contribution rules soon regarding this.

Reply:
okay